Marlin/src/HAL/SAMD51/eeprom_wired.cpp

```diff
@@ -22,7 +22,7 @@
 
 #include "../../inc/MarlinConfig.h"
 
-#if USE_WIRED_EEPROM && DISABLED(QSPI_EEPROM)
```

Comment:
Semantically speaking qspi is an hard wired eeprom, similar to any i2c eprom added on shields, but this is mounted directly on the board PCB, but yes this is really pedantry